boards/arm/thingy53_nrf5340/thingy53_nrf5340_common-pinctrl.dtsi

```diff
@@ -108,4 +108,21 @@
 		};
 	};
 
+	spi4_default: spi4_default {
+		group1 {
+			psels = <NRF_PSEL(SPIM_SCK, 0, 5)>,
+				<NRF_PSEL(SPIM_MISO, 0, 10)>,
+				<NRF_PSEL(SPIM_MOSI, 0, 9)>;
```

Comment:
Consider adding `nordic,drive-mode = <NRF_DRIVE_H0H1>;`. It may be needed for higher SCK frequencies.